IntelliTect.TestTools.WindowsTestWrapper/ExampleProjects/Notepad.AutomatedTests/Helpers/NotepadWindowHelpers.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.IO;
+using System.Windows.Forms;
+using Microsoft.VisualStudio.TestTools.UITesting;
+using Notepad.AutomatedTests.Windows;
+
+namespace Notepad.AutomatedTests.Helpers
+{
+    public class NotepadWindowHelpers
+    {
+        readonly NotepadWindow _NotepadWindow = new NotepadWindow();
+        readonly SaveAsWindow _SaveAsWindow = new SaveAsWindow();
+
+        //Ordinarily this would contain more complex operations.
+        //Use this to abstract away some of the complexity to help the test author.
+        public void SaveDocument(string location, string filename)
+        {
+            //Check if doc with same name already exists in location before saving
+            string formattedPath = location + "\\" + filename;
+            if (File.Exists(formattedPath))
+            {
+                filename += "1";
```

Comment:
It was not intentional. It was an oversight on me trying to come up with something quick and not double checking. However, your comment has got me thinking that there is a better way in general to handling saving files for a test. I will do some poking around with this...